### DISCUSSION TITLE
Multiple access to a same app

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#2760] [Q&A]
### Question

Good morning. I need to develop an application where two people can access it at the same time, and develop different processes, for example, one of them may be registering users, and the other can see what the other user is doing. Can this be developed in flet? What should I take into account to be able to do it?

Thanks

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 09u2h4n** [comment]
In flet multiple user can use one app simultaneously (In the same time).
You can use [Pubsub](https://flet.dev/docs/guides/python/pub-sub) for users see eachother that what are they doing.
The best example is [Chatting app](https://flet.dev/docs/tutorials/python-realtime-chat/)
Well you can also use a server to send users data.

**Reply by ingdesarr1** [reply]
If I need connect just two sessions, how can I do that? With a user_id or something like that?
